setup.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+#
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+echo "==================================================="
+echo "🎮 Greetings! Beginning developer-journey-app setup"
+echo "==================================================="
+
+export PROJECT_ID=${PROJECT_ID:=$(gcloud config get project)}
+export REGION=${REGION:=us-central1}
+export PROJECTNUM=${PROJECT_ID:=$(gcloud projects describe ${PROJECT_ID} --format='value(projectNumber)')}
+export REPO_OWNER=${REPO_NAME:=repo-owner}
+export REPO_NAME=${REPO_NAME:=repo-name}
+export ENV=${ENV:=dev}
+
+echo "==================================================="
+echo "🎮 Build app image"
+echo "==================================================="
+
+gcloud builds submit --project $PROJECT_ID --tag $REGION-docker.pkg.dev/$PROJECT_ID/dev-journey-repo/dev-journey .
```

Comment:
the artifact registry is created in the terraform

Reply:
After the fact. It needs to exist for the initial build and when then ci/cd gets added in. I'm adding if users don't include SiC/